API/scanner/scanner-adapter-openapi-v1.0.yaml

```diff
@@ -0,0 +1,392 @@
+openapi: 3.0.0
+info:
+  title: Harbor Scanner Adapter API
+  description: |
+    ## Overview
+
+    The API that has to be implemented in order to plug an artifact scanner in [Harbor](https://goharbor.io) registry.
```

Comment:
This API that must be implemented in order to register a new artifact scanner in [Harbor](https://goharbor.io) registry.

Reply:
This API must be implemented in order to register a new artifact scanner in Harbor registry.